src/Components/Common/SlideOver.gen.tsx

```diff
@@ -1,21 +1,22 @@
 /* TypeScript file generated from SlideOver.res by genType. */
 /* eslint-disable import/first */
 
-import * as React from "react";
+
+import * as React from 'react';
```

Comment:
gitignore `*.gen.tsx`

Reply:
This is there in gitignore but seems like its not working @bodhish

Reply:
I have merged develop as well